templates/frontend/pages/indexJournal.tpl

```diff
@@ -49,10 +49,12 @@
 			<section class="current_issue">
 				<header>
 					{strip}
-						<p class="current_issue_label">{translate key="journal.currentIssue"}</p>
-						{if $issueIdentificationString}
-							<h2 class="current_issue_title">{$issueIdentificationString|escape}</h2>
-						{/if}
+						<h2 class="current_issue_title">
+							<span class="current_issue_label">{translate key="journal.currentIssue"}</span>
+							{if $issueIdentificationString}
+						 		<span class="current_issue_identification">{$issueIdentificationString|escape}
```

Comment:
@sssoz Can you add here closing span tag?

Reply:
Oops, fixed and force-pushed.